scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -66,7 +66,7 @@
 	dateOfCollectionStart	Start	Date when the data collection started.	YYYY-MM-DD	date	62	#NAME: #VALUE 	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	dateOfCollection	citation
 	dateOfCollectionEnd	End	Date when the data collection ended.	YYYY-MM-DD	date	63	#NAME: #VALUE 	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	dateOfCollection	citation
 	kindOfData	Kind of Data	Type of data included in the file: survey data, census/enumeration data, aggregate data, clinical data, event/transaction data, program source code, machine-readable text, administrative records data, experimental data, psychological test, textual data, coded textual, coded documents, time budget diaries, observation data/ratings, process-produced data, or other.		text	64		TRUE	FALSE	TRUE	TRUE	FALSE	FALSE		citation	http://rdf-vocabulary.ddialliance.org/discovery#kindOfData
-	series	Series	Information about the Dataset series.		none	65	:	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE		citation
```

Comment:
I assume this change to citation.tsv is unintentional and should be removed.

Reply:
https://github.com/IQSS/dataverse/pull/8024/commits/f51d71eb092d59ec9adc6693454c23d8c7b5b8de